lib/hello/catalog.ex

```diff
@@ -0,0 +1,229 @@
+defmodule Hello.Catalog do
+  @moduledoc """
+  The Catalog context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Hello.Repo
+
+  alias Hello.Catalog.Product
+  alias Hello.Catalog.Category
+
+  @doc """
+  Returns the list of products.
+
+  ## Examples
+
+      iex> list_products()
+      [%Product{}, ...]
+
+  """
+  def list_products do
+    Repo.all(Product)
```

Comment:
@keliumJU what does `Repo` mean?

Reply:
Repo is a module that is used to work with database actions such as insert, update, delete, create..., depending on the database we are working with Repo uses one adapter or another for either postgresql, mysql, sqlite etc.